### PR TITLE
Update README with new badges

### PR DESCRIPTION
**Pull request by spencergilbert**
Signed-off-by: Spencer Gilbert <spencer.gilbert@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

Removed cross-build and e2e action badges
Added: react, go, docs, benchmark, fuzzy-test-on-master, CI
Grouped all CI badges on newline (wasn't entirely sure if there was a preferred order)

## Verification

Viewed local markdown rendering and on my fork

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 [![Netlify Status](https://api.netlify.com/api/v1/badges/664a5091-934c-4b0e-a7b6-bc12f822a590/deploy-status)](https://app.netlify.com/sites/thanos-io/deploys)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3048/badge)](https://bestpractices.coreinfrastructure.org/projects/3048)
 
+[![CI](https://github.com/thanos-io/thanos/workflows/CI/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3ACI)
+[![CI](https://circleci.com/gh/thanos-io/thanos.svg?style=svg)](https://circleci.com/gh/thanos-io/thanos)
+[![go](https://github.com/thanos-io/thanos/workflows/go/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3Ago)
+[![react](https://github.com/thanos-io/thanos/workflows/react/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3Areact)
+[![benchmark](https://github.com/thanos-io/thanos/workflows/benchmark/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3Abenchmark)
+[![fuzzy-test-on-master](https://github.com/thanos-io/thanos/workflows/fuzzy-test-on-master/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3Afuzzy-test-on-master)
```
Comment (bwplotka) [inline README.md:15]: what is this??????

File: README.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 [![Slack](https://img.shields.io/badge/join%20slack-%23thanos-brightgreen.svg)](https://slack.cncf.io/)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/664a5091-934c-4b0e-a7b6-bc12f822a590/deploy-status)](https://app.netlify.com/sites/thanos-io/deploys)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3048/badge)](https://bestpractices.coreinfrastructure.org/projects/3048)
 
+[![CI](https://github.com/thanos-io/thanos/workflows/CI/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3ACI)
+[![CI](https://circleci.com/gh/thanos-io/thanos.svg?style=svg)](https://circleci.com/gh/thanos-io/thanos)
+[![go](https://github.com/thanos-io/thanos/workflows/go/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3Ago)
+[![react](https://github.com/thanos-io/thanos/workflows/react/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3Areact)
+[![benchmark](https://github.com/thanos-io/thanos/workflows/benchmark/badge.svg)](https://github.com/thanos-io/thanos/actions?query=workflow%3Abenchmark)
```
Comment (bwplotka) [inline README.md:14]: What's this? :thinking:

**Comment by bwplotka** [comment]
> Grouped all CI badges on newline (wasn't entirely sure if there was a preferred order) 

Thanks for this (:

File: README.md (hunk 1 above)
Comment (spencergilbert) [reply README.md:15]: https://github.com/thanos-io/thanos/actions <- super bizarre, these are still floating around the action ui with no config in the .github folder, and haven't run for 3 months... I'll remove these two

File: README.md (hunk 2 above)
Comment (spencergilbert) [reply README.md:14]: Same as above

**Comment by spencergilbert** [comment]
@bwplotka removed the old mystery jobs :smile: don't know why github is still showing those

**Comment by squat** [comment]
re-running failed tests

**Comment by spencergilbert** [comment]
thanks - was about to look at that test

**Pull request merged**